hazelcast/src/main/java/com/hazelcast/spi/impl/NodeEngine.java

```diff
@@ -114,6 +115,13 @@
      */
     SplitBrainProtectionService getSplitBrainProtectionService();
 
+    /**
+     * Gets SQL service.
+     *
+     * @return the SQL service
+     */
+    SqlServiceProxy getSqlService();
```

Comment:
nit: `Proxy` in general is used to describe remote objects' implementation (eg. `IMap` -> `MapProxyImpl`, `ITopic` -> `ReliableTopicProxy`). Also, consider extracting an interface for `SqlService`, it will be a lot easier to run compatibility tests, if those tests need access to the `SqlService`.

Reply:
1. `SqlServiceProxy` is renamed to `SqlServiceImpl`
1. The interface part of this service is public API, which is going to be implemented in a separate ticket. So we cannot extract anything at the moment, otherwise, it will be an empty interface in the public API package. So instead I removed the method from `NodeEngine`, because we simply do not need it before the public API is implemented.